db/migrate/20171113145621_add_resource_type_and_resource_id_to_resource_monitor.rb

```diff
@@ -0,0 +1,6 @@
+class AddResourceTypeAndResourceIdToResourceMonitor < ActiveRecord::Migration[5.0]
+  def change
+    add_column :resource_monitors, :resource_type, :string
+    add_column :resource_monitors, :resource_id, :integer, index: true
```

Comment:
We've prefixed these fields with `resource_` even though we have the word resource in the model and table name. Seems a little verbose.

Reply:
I agree, it seems like we should rename `service` to `application`, and then we can have `resource_type` as `name` and then we don't need the `resource_id` field.

Reply:
Probably worth making these `null: false` as well.

Reply:
If `application` is used over `service` in the GDS world we should probably be consistent with that.

Reply:
We have `publishing_app` and `rendering_app` in the Publishing APP, so `app` might be the better choice. Although in Router we have `backend_id`...

Reply:
Ok, so we're going with `app` and `reference` ?